server/src/main/java/com/defold/extender/ExtenderUtil.java

```diff
@@ -287,10 +280,106 @@ static String getAppManifestContextString(AppManifestConfiguration manifest, Str
         return default_value;
     }
 
-    private static boolean isListOfStrings(List<Object> list) {
+    static public boolean isListOfStrings(List<Object> list) {
         return list != null && list.stream().allMatch(o -> o instanceof String);
     }
 
+    static public List<String> mergeLists(List<String> l1, List<String> l2) {
+        List<String> items = new ArrayList<>();
+        if (l1 != null) {
+            items.addAll(l1);
+        }
+        if (l2 != null) {
+            items.addAll(l2);
+        }
+        return items;
+    }
+
+    private static final String MERGE_KEY_REPLACE = "_replace";
+    static private boolean isMergeOp(String name)
+    {
+        if (name.endsWith(MERGE_KEY_REPLACE))
+            return false;
+        return true;
+    }
```

Comment:
Q: Do we have a better way than this? Better name?

Reply:
Can't think of anything right now to be honest.